gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsStatistic.java

```diff
@@ -82,10 +82,10 @@ public enum GhfsStatistic {
       TYPE_COUNTER),
   INVOCATION_COPY_FROM_LOCAL_FILE(
       StoreStatisticNames.OP_COPY_FROM_LOCAL_FILE, "Calls of copyFromLocalFile()", TYPE_COUNTER),
-  INVOCATION_CREATE(StoreStatisticNames.OP_CREATE, "Calls of create()", TYPE_COUNTER),
+  INVOCATION_CREATE(StoreStatisticNames.OP_CREATE, "Calls of create()", TYPE_DURATION),
```

Comment:
nit: should this be "Duration of.." ?

Reply:
This is pattern that is already getting used. Even otherwise the naming look ok to me.

Reply:
Yeah, but we changed it from capturing the count to capturing duration of operation. Hence we should fix the description for this too, shouldn't we?

Reply:
Made this consistent with similar existing metrics.